packages/mobile/src/send/SendConfirmation.tsx

```diff
@@ -148,6 +157,18 @@ export class SendConfirmation extends React.Component<Props, State> {
 
     const timestamp = Date.now()
 
+    CeloAnalytics.track(CustomEventNames.send_confirm, {
+      method: this.props.route.params?.isFromScan ? 'scan' : 'search',
```

Comment:
does this win for the most custom properties tracked?

Reply:
Can you elaborate?

Reply:
just the number of things being tracked is impressive.

Reply:
Yeah, we've certainly turned the analytics up a notch!